src/providers/overlayManager.tsx

```diff
@@ -72,7 +72,7 @@ export const useOverlayManager = () => {
   const incrementOverlaysOpenCount = useCallback(() => setOverlaysOpenCount((count) => count + 1), [
     setOverlaysOpenCount,
   ])
-  const decrementOverlaysOpenCount = useCallback(() => setOverlaysOpenCount((count) => (count > 0 ? count - 1 : 0)), [
+  const decrementOverlaysOpenCount = useCallback(() => setOverlaysOpenCount((count) => count - 1), [
```

Comment:
Why did we change this condition? It makes sense to not allow this to go negative right?

Reply:
oh, yeah - I set that only for testing purposes. I wanted to track how often decrement is called. I will get this condition back